lib/vcloud/launcher/launch.rb

```diff
@@ -5,11 +5,11 @@ class Launch
       class MissingPreambleError < RuntimeError ; end
       class MissingConfigurationError < RuntimeError ; end
 
-      attr_reader :config, :cli_options
+      attr_reader :config, :options
 
-      def initialize(config_file, cli_options = {})
```

Comment:
I don't have a strong opinion on this as the distinction between command-line arguments and options seems OK to me. Either way, I think that should be a separate pull request.